README.md

```diff
@@ -149,15 +149,17 @@ Optional<JSONObject> payload = sessionJson.load(persistenceKey, dataPersistence)
 
 * [Java](java/app-encryption)
 * [.NET](csharp/AppEncryption)
-* Go (coming soon!)
+* [Go](go/appencryption)
+* Service Layer (coming soon!)
 
 ### Feature Support
 
-| Feature            | Java | .NET |
-| ------------------ | ---- | ---- |
-| AWS KMS Support    | Yes  | Yes  |
-| RDBMS Metastore    | Yes  | Yes  |
-| DynamoDB Metastore | Yes  | Yes  |
+| Feature            | Java | .NET | Go  |
+| ------------------ | ---- | ---- | --- |
+| AWS KMS Support    | Yes  | Yes  | Yes |
+| RDBMS Metastore    | Yes  | Yes  | Yes |
+| DynamoDB Metastore | Yes  | Yes  | Yes |
+| SessionCache       | Yes  | Yes  | No  |
```

Comment:
I like the addition of `SessionCache` to the feature grid. We should add the [encrypt/decrypt](https://github.com/godaddy/asherah/blob/master/README.md#encrypt--decrypt) and [store/load](https://github.com/godaddy/asherah/blob/master/README.md#store--load) usage patterns as well considering Go does not yet support the latter.